lang/rs/bindings/cners/examples/loopback/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Can this move to lang/rs/bindings/examples ??

Reply:
There are examples where Rust bindings for C/C++ library ends in -sys or rs. We could rename it as well.

Reply:
Pushed changes. User can search with `cndp_binding` in Rust crates.io (whenever we publish it. This is package name). In code we will use crate name `cne_binding`. Code is kept under `lang/rs/bindings/cne`